### PR TITLE
Fix async warning, and updated lightentityfeature and colormode for HA Core 2025.1

### PR DESCRIPTION
**Pull request by kbullet**
Updated to support LightEntityFeature and color modes https://github.com/hcoohb/hass-yeelightbt/issues/64

```
2024-11-08 16:45:15.569 WARNING (MainThread) [homeassistant.components.light] Entity light.bedside (<class 'custom_components.yeelight_bt.light.YeelightBT'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <LightEntityFeature: 19> and color modes, please create a bug report at https://github.com/hcoohb/hass-yeelightbt/issues and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation

2024-11-08 16:45:15.569 WARNING (MainThread) [homeassistant.components.light] light.bedside (<class 'custom_components.yeelight_bt.light.YeelightBT'>) does not set supported color modes, this will stop working in Home Assistant Core 2025.3, please create a bug report at https://github.com/hcoohb/hass-yeelightbt/issues
```

Also updated async warning and errors https://github.com/hcoohb/hass-yeelightbt/issues/66 for 2025.1

### CODE REVIEW

File: custom_components/yeelight_bt/light.py [hunk 1]
```diff
@@ -291,6 +299,15 @@ async def async_turn_on(self, **kwargs: int) -> None:
 
         # if ATTR_EFFECT in kwargs:
         #    self._effect = kwargs[ATTR_EFFECT]
+        
+        if ColorMode.HS in self.supported_color_modes and ATTR_HS_COLOR in kwargs:
```
Comment (hcoohb) [inline custom_components/yeelight_bt/light.py:303]: Isn't this already implemented at line 266?

**Comment by hcoohb** [comment]
Thanks for the PR.

I have commented 2 additions that I think we already have in the source code.

File: custom_components/yeelight_bt/light.py (hunk 1 above)
Comment (kbullet) [reply custom_components/yeelight_bt/light.py:303]: yes you're right, I moved it out to debug an issue I was having and totally forgot about it. I've pushed an update

File: custom_components/.DS_Store
Comment (hcoohb) [inline custom_components/.DS_Store:1]: Sorry, can we remove this file?

**Review by hcoohb — COMMENTED** [review]
Thanks for the changes, can we just remove the .DS_Store file that got added in last commit?

File: custom_components/.DS_Store
Comment (kbullet) [reply custom_components/.DS_Store:1]: oh doh! it's removed.

Comment (kbullet) [reply custom_components/.DS_Store:1]: Thanks for accepting the PR.

p.s. great work btw! Did not think you were still active on this repo.

**Pull request merged**